js/src/components/paid-ads/validateAssetGroup.js

```diff
@@ -75,12 +75,13 @@ export default function validateAssetGroup( values ) {
 					? spec.lowercaseSingularName
 					: spec.lowercasePluralName;
 
-			const message = sprintf(
-				// translators: 1: The minimal number of this item. 2: Asset field name.
-				__( 'Add at least %1$d %2$s', 'google-listings-and-ads' ),
-				spec.min,
-				name
-			);
+			const format = spec.requiredSingleValue
+				? // translators: 1: (This variable is omitted). 2: Asset field name.
+				  __( 'The %2$s is required', 'google-listings-and-ads' )
+				: // translators: 1: The minimal number of this item. 2: Asset field name.
+				  __( 'Add at least %1$d %2$s', 'google-listings-and-ads' );
+
+			const message = sprintf( format, spec.min, name );
```

Comment:
```jsx
const message = spec.requiredSingleValue 
             ? sprintf(
				// translators: 1: Asset field name.
				__( 'The %1$s is required', 'google-listings-and-ads' ),
				name
			)
            :  sprintf(
				// translators: 1: The minimal number of this item. 2: Asset field name.
				__( 'Add at least %1$d %2$s', 'google-listings-and-ads' ),
				spec.min,
				name
			);
```

💅  I'd suggest this version avoiding that `(This variable is omitted)` that could be confusing for translators.

Reply:
Another option could be to revert the fields order 

```jsx
const format = spec.requiredSingleValue
				? // translators: 1 : Asset field name.
				  __( 'The %1$s is required', 'google-listings-and-ads' )
				: // translators: 1:  Asset field name. 2: The minimal number of this item
				  __( 'Add at least %2$d %1$s', 'google-listings-and-ads' );

			const message = sprintf( format, name, spec.min );
```

Reply:
Thanks for the great suggestion. Adjusted in https://github.com/woocommerce/google-listings-and-ads/pull/1908/commits/32f9f8b295ce33ab0b943e1e0c51c893c2ed1253.